scipy/stats/_continuous_distns.py

```diff
@@ -1519,20 +1519,39 @@ class exponweib_gen(rv_continuous):
 
     %(before_notes)s
 
+    See Also
+    --------
+    weibull_min, numpy.random.weibull
+
     Notes
     -----
     The probability density function for `exponweib` is:
 
     .. math::
 
-        f(x, a, c) = a c (1-\exp(-x^c))^{a-1} \exp(-x^c) x^{c-1}
+        f(x, a, c) = a c [1-\exp(-x^c)]^{a-1} \exp(-x^c) x^{c-1}
+
+    and its cumulative distribution function is:
+
+    .. math::
+
+        F(x, a, c) = [1-\exp(-x^c)]^a
+
+    for :math:`x > 0`, :math:`a > 0`, :math:`c > 0`.
 
-    for :math:`x >= 0`, :math:`a > 0`, :math:`c > 0`.
+    `exponweib` takes :math:`a` and :math:`c` as shape parameters:
 
-    `exponweib` takes :math:`a` and :math:`c` as shape parameters.
+    * :math:`a` is the exponentiation parameter,
+      with the special case :math:`a=1` corresponding to the usual
```

Comment:
Because you explicitly refer to `weibull_min` here, you can drop the word "usual" here with no loss of clarity.

Reply:
addressed, so we can merge this. also pushed a rebase